### DISCUSSION TITLE
Bringing the power of Signals to Angular Forms with Signal Forms

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#319] [Announcements]
# Bringing the power of Signals to Angular Forms with Signal Forms

Signal Forms is a new way to create forms in Angular. It's built on top of the existing `NgModel` with a similar API to `ReactiveForms`, and uses signals to create a reactive and flexible form. The main goal is to make complex forms manageable.

https://timdeschryver.dev/blog/bringing-the-power-of-signals-to-angular-forms-with-signal-forms

**Comment by jairmunoz** [comment]
Buen día,
Excelente librería, muy útil para el manejo de formulario, muchas gracias.

Me he topado con el siguiente escenario: en un formulario dinámico con arreglos hay q pasarle siempre el injector, aun usando el createFormGroup a cada createFormField()
